src/orders/app.py

```diff
@@ -139,10 +139,7 @@ def retrieve_orders(event: Dict[str, Any], context: LambdaContext) -> Dict[str,
         if is_auth is False:
             raise AuthError("User is not allow to retrieve orders")
 
-        filter_date = doorman.get_query_param_from_request(
```

Comment:
is this for delete?

Reply:
This for retrieve_orders , now using a new method get_date_param_from_request() .

Reply:
but what if we want to get another param, lets say a page or limit, when we include pagination in the backend

Reply:
In that case, you can  use doorman.get_query_param_from_request.

Reply:
In that case, you can use doorman.get_query_param_from_request.

Reply:
get_query_param_from_request is no longer coupled with a date field